EngineErrorMap.md

```diff
@@ -3286,4 +3286,8 @@ The native folder may be generated from older versions, please refer https://doc
 
 ### 16003
 
-'%s' is removed since v%s, please use '%s' instead.
\ No newline at end of file
+'%s' is removed since v%s, please use '%s' instead.
+
+### 16101
+
+The effect('%s') you are looking for does not exist, please confirm the effect name in the editor. NOTE: Since 3.6, the name of the built-in effect has been changed to its name in the editor, please check it out. More information please refer to https://docs.cocos.com/creator/manual/zh/shader/effect-inspector.html
```

Comment:
```suggestion
The effect('%s') you are looking for does not exist, please confirm the effect name in the editor. NOTE: Since 3.6, the name of the built-in effect has been changed to its name in the editor, please check it out. More information please refer to https://docs.cocos.com/creator/manual/en/shader/effect-inspector.html
```